### PR TITLE
Made Table Work

### PR DESCRIPTION
**Pull request by Saulgreene**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+CookieStore.prototype.salesPerHour = function(){
+  return Math.floor(Math.random() * ((this.range + 1) + this.minCustomers) * this.avgCookies);
+};
+
+CookieStore.prototype.populateHourlyCount = function() {
+  for (var i = 0; i < openHours.length; i++) {
+    this.hourlyCount.push(this.salesPerHour());
+  }
+};
+CookieStore.prototype.makeRow = function() {
```
Comment (bharding2) [inline app.js:36]: need some white space between 35 and 36 for consistency

File: app.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+  storeRow.appendChild(storeName);
+  storeName.textContent = this.displayName;
+
+  for (var i = 0; i < this.hourlyCount.length; i++) {
+    var hourlyCookieCount = document.createElement('td');
+    storeRow.appendChild(hourlyCookieCount);
+    hourlyCookieCount.textContent = this.hourlyCount[i];
+  }
+};
+function displayTimes(){
```
Comment (bharding2) [inline app.js:50]: need white space between 49 and 50 for consistency

File: index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+          <label for="maxCustomers">Maximun Customers/Hour:</label>
+          <input type="text" name="maxCustomers">
+          <label for="avgCookies">Average Cookies/Customer:</label>
+          <input type="text" name="avgCookies">
+          <button type="submit">Submit</button>
+        </fieldset>
+      </form>
+    </div>
+
+
```
Comment (bharding2) [inline index.html:40]: too many white spaces!  one is fine :)

File: style.css [hunk 4]
```diff
@@ -0,0 +1,32 @@
+body {
+  background-color: antiquewhite;
+}
+  .top {
```
Comment (bharding2) [inline style.css:4]: lines 4-13 appear to be indented an extra level

File: app.js [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+var storeOne = new CookieStore('first-and-pike', 'First and Pike', 23, 65, 6.3);
+var storeSeaTac = new CookieStore('seatac-airport', 'Seatac Airport', 3, 24, 1.2);
+var storeSeattle = new CookieStore('seattle-center', 'Seattle Center', 11, 38, 3.7);
+var storeCapitol = new CookieStore('capitol-hill', 'Capitol Hill',20, 38, 2.3);
+var storeAlki = new CookieStore('alki', 'Alki', 2, 16, 4.6);
+
+//stores Array
+var stores = [storeOne, storeSeaTac, storeSeattle, storeCapitol, storeAlki];
+
+function CookieStore(name, displayName, minCustomers, maxCustomers, avgCookies){
```
Comment (fraziermork) [inline app.js:17]: Great!

File: index.html [hunk 6]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Salmon Cookies</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <header class = "top">
+    <h1>Salmon Cookies</h1>
+    			<p id= "title">cookies to go with your coffee</p>
```
Comment (fraziermork) [inline index.html:10]: Be careful about your indentation! Also, don't put spaces on either side of an equals sign in html.

File: style.css [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+    color: black;
+    background-color: pink;
+    text-align: center;
+    border:0px;
+    font-size: 40px;
+    padding: 10px;
+    margin-right: 10px;
+    margin-left: 10px;
+  }
+  #title {
```
Comment (fraziermork) [inline style.css:14]: Try to prefer styling things with classes instead of ids.

File: app.js [hunk 8]
```diff
@@ -0,0 +1,169 @@
...
+  }
+}
+
+function makeAllRows() {
+  for (var i = 0; i < stores.length; i++){
+    stores[i].makeRow();
+  }
+}
+
+displayTimes();
```
Comment (fraziermork) [inline app.js:75]: You could even break these three lines into another function, like 'drawTable' or something!

File: app.js [hunk 9]
```diff
@@ -0,0 +1,169 @@
...
+};
+function displayTimes(){
+  for (var ii = 0; ii < openHours.length; ii++){
+    var timeOfDay = document.createElement('th');
+    timeOfDay.textContent = openHours[ii];
+    table.appendChild(timeOfDay);
+  }
+}
+
+var locations = 'Locations:';
```
Comment (fraziermork) [inline app.js:58]: These lines should be broken out into a function too.

**Review by fraziermork — COMMENTED** [review]
Nice work!

**Pull request merged**